src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml.cs

```diff
@@ -127,7 +127,7 @@ private void ExecuteUserAction(UserAction action, NuGetActionType actionType)
                     nugetUi.DependencyBehavior = model.Options.SelectedDependencyBehavior.Behavior;
                     nugetUi.RemoveDependencies = model.Options.RemoveDependencies;
                     nugetUi.ForceRemove = model.Options.ForceRemove;
-                    nugetUi.Projects = model.GetSelectedProjects(action);
+                    // nugetUi.Projects = model.GetSelectedProjects(action);
```

Comment:
commented code

Reply:
This one will have to remain until we can update the NuGetUI, which we cannot until the Actions change.